.travis.yml

```diff
@@ -12,7 +12,7 @@ jobs:
   include:
     - stage: 'Test'
       name: 'Run code linting and test'
-      script: yarn lint && yarn ci
+      script: yarn ci
```

Comment:
Honestly, I don't think this is the right choice.
Here's the only place in the pipeline where `eslit` checks the codebase, otherwise any liniting error can slip in our code.

Reply:
I think this is way too far in the pipeline to catch linting errors. We should use husky or something for that instead. The build job in this PR shouldn't fail if an untouched file suddenly starts producing linting errors.

Reply:
Actually this practice (running the linter in the pipeline) is followed in all the services we are developing at the moment, and I'd like to follow it here as well. 
Definitely, there might be a reason why the test failed in this PR, I'd rather look more into the issue.

Reply:
Fair enough, I put the linting back in and fixed the linter errors.